src/Params/UserReportParams.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Avtocod\B2BApi\Params;
+
+final class UserReportParams
+{
+    /**
+     * Report unique ID (e.g.: `some_report_uid_YV1KS9614S107357Y@domain`).
+     *
+     * @var string
+     */
+    private $report_uid;
+
+    /**
+     * True, if necessary include report content into response.
+     *
+     * @var bool
+     */
+    private $include_content = true;
+
+    /**
+     * True, if necessary detailed information about report.
+     *
+     * @var bool
+     */
+    private $detailed = true;
```

Comment:
@tarampampam @eldario

Similar pairs of getters/setters can be taken out **to traits**. But, I doubt... 🤔

Reply:
So far without using traits...